cmd/cluster-init/update_bootstrap_secrets.go

```diff
@@ -18,6 +18,7 @@ const (
 	testCredentials     = "test-credentials"
 	kubeconfig          = "kubeconfig"
 	config              = "config"
+	token               = "token.txt"
```

Comment:
can this be moved to `main.go` since that is the only usage?

Reply:
Done.